pandokia/helpers/dict_comp.py

```diff
@@ -153,7 +153,7 @@ def read_reference(fn):
 ##########
 
 
-spaces = re.compile("[\s]")
+spaces = re.compile(r"[\s]")
```

Comment:
This is also never get referenced after this.

Reply:
Since `spaces` is unused I would recommend gutting it.

Reply:
How sure are you?  Are you using `grep`?  And even if it is not used in pandokia, please check whether it is a namespace global that can be imported by another caller `from dict_comp import spaces`

Reply:
I was using grep in pandokia repo.
But yes, I don't want to take it out since I don't want if how other applications use pandokia.

Reply:
I first searched for usages for the variable using PyCharm and nothing came up. I then went to the extreme and searched for "spaces" across all projects (ignoring comments) and found one other instance in test_thisdir.py which has its own definition and usage of a `spaces` variable.

Reply:
Ah, I wasn't aware that other projects outside of ours use it. Maybe leave it then.